examples/nonLinearStaticCantileverAero/DiffEq.jl

```diff
@@ -0,0 +1,89 @@
+# Include libreries
+using BoundaryValueDiffEq, Plots, FileIO
+# Register elapsed time
+tiempo = @elapsed begin
```

Comment:
you're probably only interested to time line 49 ie. the call to the solve function

Reply:
Done